cg/meta/workflow/rnafusion.py

```diff
@@ -129,58 +129,21 @@ def write_samplesheet(self, case_id: str, strandedness: str, dry_run: bool = Fal
                 config_path=self.get_case_config_path(case_id=case_id),
             )
 
-    def write_params_file(
-        self, case_id: str, genomes_base: Optional[Path] = None, dry_run: bool = False
-    ) -> None:
-        """Write params-file for rnafusion analysis in case folder."""
-        default_options: Dict[str, str] = self.get_default_parameters(case_id=case_id)
-        if genomes_base:
-            default_options["genomes_base"] = genomes_base
-        LOG.info(default_options)
-        if dry_run:
-            return
-        NfBaseHandler.write_nextflow_yaml(
-            content=default_options,
-            file_path=self.get_params_file_path(case_id=case_id),
-        )
-
-    def write_trailblazer_config(self, case_id: str, tower_id: str) -> None:
-        """Write Tower IDs to a .YAML file used as the trailblazer config."""
-        config_path = self.get_trailblazer_config_path(case_id=case_id)
-        LOG.info(f"Writing Tower ID to {config_path.as_posix()}")
-        WriteFile.write_file_from_content(
-            content={case_id: [tower_id]},
-            file_format=FileFormat.YAML,
-            file_path=config_path,
-        )
+    def get_pipeline_parameters(self, case_id: str, genomes_base: Optional[Path] = None) -> dict:
```

Comment:
I think it would be better if this method returns `RnafusionParameters`. If you end up needing a dictionary somewhere else, you can convert it there. It's always easier to work with data models than dicts

Reply:
I think this method should return something general that can be use in other pipelines (e.g. taxprofiler). I will make then a parent class (PipelineParameters) and return that

Reply:
Yes, that's a good idea, a data model with common fields between pipelines 💯

Reply:
done!